scripts/gen_test_repos.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Build the test repos to be used by Pulp Smash.
+#
+# This script outputs to the /test_repos/ folder. The built assets should
+# never be checked into source control, so that directory is ignored
+# using .gitignore.
+#
+# This script is primarily used by a Jenkins job which builds the assets and
+# copies the contents of /test_repos/ to [0].
+#
+# [0]:  https://repos.fedorapeople.org/repos/pulp/pulp/test_repos
+#
+# NOTE: This script should be run from the repository root directory. That is,
+# this script should be run from this script's parent directory.
+
```

Comment:
A good technique for making more reliable scripts is to add:

```sh
set -euo pipefail
```

Those options will cause the script to execute under a much more strict set of conditions. See [this blog post](http://redsymbol.net/articles/unofficial-bash-strict-mode/).